server/src/app.ts

```diff
@@ -1,6 +1,7 @@
 import { join } from "path";
 import AutoLoad, { AutoloadPluginOptions } from "@fastify/autoload";
 import { FastifyPluginAsync } from "fastify";
+const mongoose = require("mongoose");
```

Comment:
can we keep the import method consistent between all imports? e.g import x from '@x'